demo/demo.gradle

```diff
@@ -131,4 +134,32 @@ if (canQuarkus) {
             buildImage = "quay.io/quarkus/ubi-quarkus-native-image:21.1.0-java11"
         }
     }
+}
+
+def grpcApiRepository = project.property('grpcApiRepository')
+def grpcApiTag = project.version
+
+def webRepository = project.property('webRepository')
+def webTag = project.version
+
+def tagGrpcApiTask = tasks.register('tagGrpcApi', DockerTagImage) {
+    DockerTagImage t ->
+        t.imageId = 'deephaven/grpc-api:local-build'
+        t.repository = grpcApiRepository
+        t.tag = grpcApiTag
+        t.dependsOn project(':grpc-api-server-docker').tasks.findByName('dockerBuildImage')
+}
+
+def tagWebTask = tasks.register('tagWeb', DockerTagImage) {
+    DockerTagImage t ->
+        t.imageId = 'deephaven/web:local-build'
+        t.repository = webRepository
+        t.tag = webTag
+        t.dependsOn project(':web-client-ide').tasks.findByName('buildDocker')
```

Comment:
hm... it would be onerous to have multiple tasks to build the same image w/ different web-version.  Think we could just route a `-PwebVersion=1.2.3` into the one-and-only-task, so whoever invokes `pushAll` can do `pushAll -PwebVersion=forDemoVersion` ?